cfg/config.go

```diff
@@ -42,12 +42,12 @@ func loadEnvVar() error {
 	}
 
 	if err := scanner.Err(); err != nil {
-		return err
+		return fmt.Errorf("error while scaning .env file: %w", err)
 	}
 
 	for _, l := range lines {
 		pair := strings.Split(l, "=")
-		if len(pair) != 2 {
+		if len(pair) != 2 { //nolint:gomnd // Its not magic number
```

Comment:
Please make a const instead of linter ignore command

Reply:
done